### DISCUSSION TITLE
Keycloak operator (KO) -> keycloak deployment on k8s without OLM (operator lifecycle manager)

### DISCUSSION THREAD

**Discussion by jankohout95** [keycloak/keycloak#11267] [Q&A]
Hey guys from maintenance team, is it possible to deploy KO with all CRDs on k8s cluster without OLM? I have found only this up to date deployment information: 
![image](https://user-images.githubusercontent.com/100849407/163131424-a821126d-5f2e-4dfa-9df2-70e84557404b.png)
Source: https://operatorhub.io/operator/keycloak-operator

Is there any other easy way as it is described in OLM but deploy KO without OLM?

We would like to lock keycloak on v17 alongside with KO on our cluster and we are not sure how OLM works, if it is gonna continuously update versions, etc.... (It is even "big hammer to hit the small nail", because we use just one operator -> KO, so we think OLM is not needed in this case...)

**Comment by GretUp** [comment]
I think you can: https://www.keycloak.org/docs/latest/server_installation/index.html#_install_by_command
Make sure to use the new operator: https://github.com/keycloak/keycloak/tree/main/operator

When I read the documentation it is for me not clear how to proceed. As usual, keycloak documentation is poor.
